### ISSUE TITLE
Project-less mode

### ISSUE THREAD

**Issue by morevnaproject** [morevnaproject-org/RenderChan#72]
Allow Renderchan to work in situation when no "project.conf" file found. This mode will have a limited functionality (for example, as there are no project resolution defined, it will use native resolution of a file).

Introducing this mode will expand use cases, will make RenderChan friendly for new users, and simplify integration into other tools (I plan to integrate it into Synfig, for example).

**Comment by morevnaproject** [comment]
I suggest to use following schema to define location of rendered files in project-less mode:

For given file path:
/path/to/some/file.blend

Rendering path will be:
/path/to/some/render/file.blend.png

For rendering format I suggest to use "png", as it is most commonly supported by all apps and provides support for transparency.

**Comment by morevnaproject** [comment]
@scribblemaniac maybe you will be interested to implement this feature? ^__^

**Comment by scribblemaniac** [comment]
I agree that this is a good idea for improving user friendliness. Even I have to look up things each time I want to start a new project just to know how to write the project.conf file. Alternatively there could be a way to easily or automatically generate a default project.conf file.

When running RenderChan, it could accept arguments for the options in the conf file. If there's no conf file it could use those arguments or default values if not provided, and if there is a conf file those could be used as overrides.

Unfortunately I will not have very much time to dedicate to open-source projects over the next few months, and even if I did, I don't think this is the first thing I'd be want to work on for RenderChan.

**Comment by morevnaproject** [comment]
Got it! In any case, thank you for taking your time to write your thoughts! ^__^
